x-pack/plugins/reporting/public/lib/job.tsx

```diff
@@ -0,0 +1,269 @@
+        values: { attempts: this.attempts, max_attempts: this.max_attempts },
+      });
+    } else if (this.getWarnings()) {
+      smallMessage = i18n.translate('xpack.reporting.jobStatusDetail.warningsText', {
+        defaultMessage: 'See report info for warnings.',
+      });
+    } else if (this.getError()) {
+      smallMessage = i18n.translate('xpack.reporting.jobStatusDetail.errorText', {
+        defaultMessage: 'See report info for error details.',
+      });
+    }
+
+    if (smallMessage) {
+      return (
+        <EuiText size="s">
+          <EuiTextColor color="subdued">{smallMessage}</EuiTextColor>
+        </EuiText>
+      );
+    }
+
+    return null;
+  }
+
+  getStatus() {
+    const statusLabel = jobStatusLabelsMap.get(this.status) as string;
+    const statusTimestamp = this.getStatusTimestamp();
+
+    if (statusTimestamp) {
+      return (
+        <FormattedMessage
+          id="xpack.reporting.jobStatusDetail.statusTimestampText"
+          defaultMessage="{statusLabel} at {statusTimestamp}"
+          values={{
+            statusLabel,
+            statusTimestamp: (
+              <span className="eui-textNoWrap">{this.formatDate(statusTimestamp)}</span>
+            ),
+          }}
+        />
+      );
+    }
+
+    return statusLabel;
+  }
+
+  getStatusLabel() {
+    return (
+      <>
+        {this.getStatus()}
+        {this.getStatusMessage()}
+      </>
+    );
+  }
+
+  getCreatedAtLabel() {
+    if (this.created_by) {
+      return (
+        <>
+          <div>{this.formatDate(this.created_at)}</div>
+          <span>{this.created_by}</span>
+        </>
+      );
+    }
+    return this.formatDate(this.created_at);
+  }
+
+  /*
+   * We use `output.warnings` to show the error of a failed report job,
+   * and to show warnings of a job that completed with warnings.
+   */
+
+  // There is no error unless the status is 'failed'
+  getError() {
+    if (this.status === FAILED) {
+      return this.warnings;
+    }
+  }
+
+  getWarnings() {
+    if (this.status !== FAILED) {
+      const warnings: string[] = [];
+      if (this.isDeprecated) {
+        warnings.push(
+          i18n.translate('xpack.reporting.jobWarning.exportTypeDeprecated', {
+            defaultMessage:
+              'This is a deprecated export type. Automation of this report will need to be re-created for compatibility with future versions of Kibana.',
+          })
+        );
+      }
+      if (this.csv_contains_formulas) {
+        warnings.push(
+          i18n.translate('xpack.reporting.jobWarning.csvContainsFormulas', {
+            defaultMessage:
+              'Your CSV contains characters which spreadsheet applications can interpret as formulas.',
+          })
+        );
+      }
+      if (this.max_size_reached) {
+        warnings.push(
+          i18n.translate('xpack.reporting.jobWarning.maxSizeReachedTooltip', {
+            defaultMessage: 'Max size reached, contains partial data.',
```

Comment:
Can you rewrite as something like this:

Your search reached the max size and contains partial data.